src/datachain/progress.py

```diff
@@ -132,8 +133,24 @@ def format_dict(self):
         return d
 
 
-class CombinedDownloadCallback(TqdmCallback):
+class CombinedDownloadCallback(Callback):
     def set_size(self, size):
         # This is a no-op to prevent fsspec's .get_file() from setting the combined
         # download size to the size of the current file.
         pass
+
+    def increment_file_count(self, n: int = 1) -> None:
+        pass
+
+
+class TqdmCombinedDownloadCallback(CombinedDownloadCallback, TqdmCallback):
```

Comment:
I have modified the callback to also show file counts on prefetching.
This will not show up on pytorch however.

Eg:
```
Download: 1.03MB [00:01, 605kB/s, 50 files]
```